client/e2e/pages/seasonsPage.ts

```diff
@@ -0,0 +1,165 @@
+import {type Locator, type Page, expect} from '@playwright/test';
+import {EventsPage} from '../pages/EventsPage';
+import { EventsInfo2 } from '../testData/ConstsPackage';
+
+export class SeasonsPage {
+  readonly page: Page;
+
+  // Homepage
+  readonly emailButton: Locator;
+  readonly manageTicketingButton: Locator;
+  readonly dashboardSeasonsButton: Locator;
+
+  // ../ticketing
+  readonly SeasonHeading: Locator;
+  readonly addSeasonButton: Locator;
+
+  readonly pageHeader: Locator;
+  readonly seasonNameBlank: Locator;
+  readonly seasonStartDate: Locator;
+  readonly seasonEndDate: Locator;
+  readonly imageURL: Locator;
+
+  readonly seasonOption1: Locator;
+  readonly seasonOption2: Locator;
+
+  readonly seasonEdit: Locator;
+  readonly seasonSave: Locator;
+  readonly seasonDelete: Locator;
+
+  readonly seasonContinue: Locator;
+
+  readonly DashboardButton: Locator;
+  
+  readonly addEvent: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.emailButton= page.getByText(process.env.TEST_EMAIL as string);
+
+    this.manageTicketingButton=page.getByText('Manage Ticketing').first();
+    this.dashboardSeasonsButton=page.getByRole('button', {name: 'Seasons'});
+
+    this.pageHeader = page.getByRole('heading', { name: 'Select Season' });
+
+    this.addSeasonButton = page.getByRole('button', { name: 'Add Season' });
+
+    this.seasonNameBlank = page.getByLabel('Season Name:');
+    this.seasonStartDate = page.getByLabel('Start Date:');
+    this.seasonEndDate = page.getByLabel('End Date:');
+    this.imageURL=page.getByLabel('Image URL:');
+
+    this.seasonOption1=page.getByLabel('Use Default Image');
+    this.seasonOption2=page.getByLabel('Active');
+
+    this.seasonEdit=page.locator(".flex[data-mui-internal-clone-element=true]").first();
+    this.seasonSave=page.getByRole('button', {name: 'Save'});
+    this.seasonDelete=page.locator(".flex[data-mui-internal-clone-element=true]").last();
+
+    this.seasonContinue=page.getByRole('button', { name: 'Continue' });
+
+    this.addEvent=page.getByRole('button', { name: 'Add event' });
+
+  }
+
+  async addNewSeason()
+  {
+    await this.addSeasonButton.click();
+    await this.seasonNameBlank.fill('TestName');
+    await this.seasonStartDate.fill('2020-02-02');
+    await this.seasonEndDate.fill('2020-03-02');
+    await this.imageURL.fill('https://www.hindustantimes.com/ht-img/img/2023/08/25/550x309/international_dog_day_1692974397743_1692974414085.jpg');
```

Comment:
I suggest building an interface in ConstPackages to contain the season data.  This will enable a future improvement to create a version with unique identifiers and reduce the risk of flaky/failed tests interfering with other tests.

Also, set up the page object functions to accept arguments for the various season/ticket information instead of hardcoding them.  Then pass that information from the test file.  This increases your flexibility by allowing page functions to be reused in other tests.